jenkins/tests/test_bokchoy_db_pull_request.py

```diff
@@ -28,7 +28,7 @@ class BokchoyPullRequestTestCase(TestCase):
            return_value=None)
     @patch('jenkins.bokchoy_db_pull_request.create_pull_request',
            return_value=None)
-    @patch('jenkins.bokchoy_db_pull_request.delete_branch',
+    @patch('jenkins.github_helpers.delete_branch',
```

Comment:
How were these tests passing before?

Reply:
I was doing a relative import into that file so the patch was valid, and it wasnt being called in the test so the assertion passed as well.